app/views/ui-components/v1/cards/_metal_level_select.html.slim

```diff
@@ -259,36 +262,49 @@ javascript:
         if (plan.network_information)
           network = `NETWORK NOTES <a data-toggle="tooltip" data-placement="top" data-container="body" title="${plan.network_information}"><i class="fas fa-question-circle"></i></a>`
           var innerHTML =
-          `<div class="col-xs-4 reference-plans">
+          `<div class="col-xs-4 reference-plans plan-tile">
             <div class="col-xs-12 p0 mb-1">
               <label class="container">
-                <p class="heading-text reference-plan-title mb-1"> ${plan.title}</p>
-                <span class="plan-label">Type:</span> <span class="rp-plan-info" id="plan-type-value">${plan.product_type}</span><br>
-                <span class="plan-label"">Carrier:</span> <span class="rp-plan-info" id="plan-carrier-value">${plan.carrier_name}</span><br>
-                <span class="plan-label">Level:</span> <span class="rp-plan-info" id="plan-level-value">${plan.metal_level_kind}</span><br>`;
+                <p class="heading-text reference-plan-title mt-1"> ${plan.title}</p>
+                <div class="row mt-n2">
+                  <div class="${(plan.is_pvp_eligible && isPvpFeatureEnabled) ? 'pvp-ref-plan-indicator' : 'hidden'}">
+                    <i class="fas fa-lg fa-square" data-toggle="tooltip" data-placement="top" data-container="body" title="${pvpTooltipValue}"></i> Premium Value Plan
+                  </div>
+                  <div class="${plan.is_standard_plan ? (isPvpFeatureEnabled ? 'standard-ref-plan-indicator' : 'text-uppercase') : 'hidden'} ml-1" style="${isPvpFeatureEnabled === 'true' ? '' : 'color:green'}">
```

Comment:
`style="${isPvpFeatureEnabled === 'true' ? '' : 'color:green'}"` should be checking boolean now, like
`style="${isPvpFeatureEnabled ? '' : 'color:green'}"`, right?